src/index.html

```diff
@@ -2,11 +2,46 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length_30 calendar--start-day_sat calendar--page">
+      <div class="calendar__day-cell"></div>
```

Comment:
Why do need modifier here?
```suggestion
      <div class="calendar__day"></div>
```